spec.md

```diff
@@ -2232,6 +2244,15 @@ The following table shows what the Plugin SHOULD return when receiving a second
 | MULTI_NODE     | OK (idempotent) | ALREADY_EXISTS | OK                  | OK                 |
 | Non MULTI_NODE | OK (idempotent) | ALREADY_EXISTS | FAILED_PRECONDITION | FAILED_PRECONDITION|
 
+NOTE: If the Plugin supports the `SINGLE_NODE_MULTI_WRITER` capability, use the following table instead for what the Plugin SHOULD return when receiving a second `NodePublishVolume` on the same volume on the same node:
+
+|                                | T1=T2, P1=P2    | T1=T2, P1!=P2  | T1!=T2, P1=P2       | T1!=T2, P1!=P2      |
+|--------------------------------|-----------------|----------------|---------------------|---------------------|
+| SINGLE_NODE_SINGLE_WRITER      | OK (idempotent) | ALREADY_EXISTS | FAILED_PRECONDITION | FAILED_PRECONDITION |
+| Non SINGLE_NODE_SINGLE_WRITER  | OK (idempotent) | ALREADY_EXISTS | OK                  | OK                  |
```

Comment:
Did `Non SINGLE_NODE_SINGLE_WRITER` meant -> `SINGLE_NODE_MULTI_WRITER` ? We are saying above that - this table applies for `SINGLE_NODE_MULTI_WRITER` but we don't use that word in the table - so it is bit confusing.

Reply:
Yeah, I went back and forth on the wording here. I had a row for `SINGLE_NODE_MULTI_WRITER` but realized I would probably also need a row for "Everything else" but it would be treated in the same way as `SINGLE_NODE_MULTI_WRITER` so I combined them into the "Non" row. I think having a row for this to make it explicit is a good idea, I'll add this.

Reply:
Done